docs/user_guide/validation/data-validation.rst

```diff
@@ -46,9 +53,12 @@ Flagged datapoints are skipped for lower severity warnings in the same criteria
 item (e.g.: if datapoints are flagged for the ``error`` level, they will not be
 checked again for ``low``).
 
-The second criteria item (for variable *Primary Energy|Coal*) uses the old notation.
-Its use is deprecated for being more verbose (requires each warning level to be
-a separate criteria item) and slower to process.
+Validation arguments that are identical for all warning levels can be set once as part
+of the filter arguments. This is illustrated in the second criteria item.
```

Comment:
I would spell out the example more explicitly. Something like:
The resulting criteria are, lower_bound: 0 and upper_bound: 5 with warning level error and lower_bound: 0 and upper_bound: 2.5 with warning level low.

Reply:
Thanks, changed the examples to be more explicit and useful...